### ISSUE TITLE
[Tutorial 7] Is a false negative a false alarm?

### ISSUE THREAD

**Issue by studersi** [dune73/test-4#34]
>**False alarms are possible in both directions. Attacks that are not detected are called false negatives.** The Core Rules are strict and careful to keep the number of false negatives low. An attacker needs to possess a great deal of savvy to circumvent the system of rules, especially at higher paranoia levels. Unfortunately, this strictness also results in alarms being triggered for normal requests. It is commonly the case that at a low degree of separation, you either get a lot of false negatives or a lot of false positives. Reducing the number of false negatives leads to an increase in false positives and vice versa. Both correlate highly with one another.

Is a false negative really considered a false alarm? No alarm is raised in such a case.

**Comment by dune73** [comment]
Correct. It's not a false alarm. 

How do we call it then?

Maybe reword it around "Errors when examining requests are possible in two directions."
